py/client/pydeephaven/_console_service.py

```diff
@@ -32,7 +32,7 @@ def start_console(self):
                 except Exception as e:
                     raise DHError("failed to start a console.") from e
 
-    def run_script(self, server_script: str) -> Any:
+    def run_script(self, server_script: str, systemic: bool = False) -> Any:
         """Runs a Python script in the console."""
```

Comment:
1. This seems to be for internal use only, but is this something users might need? 
2. There is any test for when `systemic` is set to True. If we can't easily verify the result, maybe at least we can be sure 'True' isn't breaking anything. Ignore the comment, if we have a test in Core+.
3. Document the parameter `systemic`